x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec

```diff
@@ -1477,3 +1467,932 @@ ROW salary = 12300, emp_no = 5, gender = "F"
 emp_no:integer
 5
 ;
+///////////////////////////
```

Comment:
Almost all the tests below have been ported from the same set of `stats` tests with filters and adapted to `inlinestats`.